app/views/webhooks/_message_status.njk

```diff
@@ -0,0 +1,16 @@
+{% set statusStyleMap = {
+"PENDING": "govuk-tag--grey",
+"SUCCESSFUL": "govuk-tag--green",
+"FAILED": "govuk-tag--red",
+"WILL_NOT_SEND": "govuk-tag--yellow"
+} %}
+{% set statusTextMap = {
+"PENDING": "PENDING",
```

Comment:
Should the labels be upper-case? I would have thought we’d use title (or maybe title) case and use CSS to make it all caps.

Reply:
Yeah CSS already makes it all caps by virtue of it being a tag, the result will be the same, happy with either